client/login/magic-login/index.jsx

```diff
@@ -359,6 +358,25 @@ class MagicLogin extends Component {
 			return null;
 		}
 
+		if ( this.props.isJetpackLogin ) {
```

Comment:
Do we need the additional check (`this.props.isFromAutomatticForAgenciesPlugin`) that was previously on line 1366?

Reply:
Good point, thanks. Fixed in b3f6519